### PR TITLE
Add WG Slack Channel 

### PR DESCRIPTION
**Pull request by arbiv**
Added the WG slack channel to the readme

### CODE REVIEW

File: wgs/cnai/onboarding/index.md [hunk 1]
```diff
@@ -13,11 +13,12 @@ Start by reading the [white paper](https://tag-runtime.cncf.io/whitepapers/cloud
...
 ## Step 2: Check the Most Recent Posts on the WG Blog
 
-Review the [most recent posts](https://tag-runtime.cncf.io/blog/) on the Cloud-Native AI blog for updates and announcements. For example:
+Review the [most recent posts](https://tag-runtime.cncf.io/blog/) on the Cloud-Native AI blog for updates and announcements.
 
 ## Step 3: Explore Previous Discussions
 
-Look into the [notes](https://docs.google.com/document/d/1tS6gL5qjGLaHDtAChssqJhBVm8BIpopUA3Ddzm9LHwg/edit) from previous discussions to get a sense of the group's activities and interests. Videos are also available for a deeper dive, if desired.
+* Look into the [notes](https://docs.google.com/document/d/1tS6gL5qjGLaHDtAChssqJhBVm8BIpopUA3Ddzm9LHwg/edit) from previous discussions to get a sense of the group's activities and interests. Videos are also available for a deeper dive, if desired.
+* Check the latest discussions in the #wg-artificial-intelligence channel on [CNCF Slack](https://slack.cncf.io/).
```
Comment (zanetworker) [inline wgs/cnai/onboarding/index.md:21]: Can we make it more specific to point the channel itself: https://cloud-native.slack.com/archives/C05TYJE81SR ?

Comment (raravena80) [reply wgs/cnai/onboarding/index.md:21]: @arbiv following up on this. Can you make the change? Thank you!

Comment (raravena80) [reply wgs/cnai/onboarding/index.md:21]: @arbiv ping.

Comment (arbiv) [reply wgs/cnai/onboarding/index.md]: @raravena80 @zanetworker Sorry, i missed this thread. Applied the suggested changes

**Review by raravena80 — APPROVED** [review]

**Pull request merged**